__tests__/app.test.js

```diff
@@ -283,6 +283,39 @@ describe("GET: /api/articles should return an array of article objects sorted by
         expect(msg).toBe("404: Topic not found");
       });
   });
+  test("200: should return an array of objects that are sorted by a query ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=votes")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body;
+        expect(articles.length).toBe(11);
+        expect(articles).toBeSortedBy("votes", { descending: true });
+      });
+  });
+
+  test("200: should return an array of objects that are sorted by a query and order", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=votes&order=asc")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body;
+        expect(articles.length).toBe(11);
+        expect(articles).toBeSortedBy("votes");
+      });
+  });
+
+  test("404: should return a bad request if sort_by column doesn't exist", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=chicken&order=asc")
+      .expect(404)
+      .then((response) => {
+        const {
+          body: { msg },
+        } = response;
+        expect(msg).toBe("404: Column not found");
+      });
+  });
 });
 
 describe("POST /api/articles/:article_id/comments should create a comment and add it to database", () => {
```

Comment:
I'd consider adding the following tests here to ensure that all of your endpoint behave as expected. 
 
  -  Status 400. invalid `sort_by` query, e.g. `?sort_by=chicken` - this one is currently 404
  - Status 400. invalid `order` query, e.g. `?order=bananas`
  - Status 200. valid `topic` query, but has no articles responds with an empty array of articles, e.g. `?topic=paper`